app/views/backend/tax_declarations/index.html.haml

```diff
@@ -11,4 +11,5 @@
       - export_params = params.slice(:period, :state)
       - d.item :export_as_pdf.tl, params: export_params.merge(format: :pdf)
       - d.item :export_as_csv.tl, params: export_params.merge(format: :csv)
+
```

Comment:
Can you remove the blank line?

Reply:
I think the line adds to readability here.